content/api/api-elements.md

```diff
@@ -283,7 +283,7 @@ The following response body found 2 elements with the **elementType** `EC2`.  Yo
 
 | Parameter | Parameter Type | Data Type | Description |
 |-----------------|----------------|-----------|-------------|
-| elementTemplate | Body | JSON | ?? |
+| elementTemplate | Body | JSON | See [here](/capacity-monitoring/inventory/inventory-element-display-name) |
```

Comment:
The CURL section directly below this one already has a link to the page you've linked here. I would just put a brief description, maybe something like:
```suggestion
| elementTemplate | Body | JSON | A naming template to preview using an element of your choice. |
```